content/reference-rippled.md

```diff
@@ -513,12 +514,15 @@ API methods for the Websocket and JSON-RPC APIs are defined by command names, an
 ## List of Public Commands ##
 
 * [`account_currencies` - Get a list of currencies an account can send or receive](#account-currencies)
+* [`account_channels` - Get a list of payment channels where the account is the source](#account-channels)
 * [`account_info` - Get basic data about an account](#account-info)
 * [`account_lines` - Get info about an account's trust lines](#account-lines)
 * [`account_objects` - Get all ledger objects owned by an account](#account-objects)
 * [`account_offers` - Get info about an account's currency exchange offers](#account-offers)
 * [`account_tx` - Get info about an account's transactions](#account-tx)
 * [`book_offers` - Get info about offers to exchange two currencies](#book-offers)
+* [`channel_authorize` - Sign a claim for money from a payment channel](#channel-authorize)
+* [`channel_verify` - Check a payment channel claim's signature](#channel-verify)
```

Comment:
This is probably a style discussion to have elsewhere, but I disagree on the possessive. As a student of Latin, whose genitive case is quite flexible (and ultimately quite similar English's possessive, partitive genitive aside), I'm inclined to use whichever form conveys the intended information most succinctly. In this case, "Check the payment channel claim's signature" reads more clearly than "Check the signature of the payment channel claim" to me so I say we keep it.